### DISCUSSION TITLE
How to do simulcast with aiortc ?

### DISCUSSION THREAD

**Discussion by lamhoangtung** [aiortc/aiortc#677] [Q&A]
Thanks for the great work. I was wondering if simulcast can be done with aiortc. If yes, can you give a little guide on how to do so, especially a sample with Janus gateway would be great

Thanks for checking by ^^

**Comment by jlaine** [comment]
No currently simulcast is not implemented, but PRs are welcome :)

**Reply by lamhoangtung** [reply]
Can you give a little guide on how to do so @jlaine ? I would love to submit a PR for it

**Reply by jlaine** [reply]
I honestly have not research how simulcast works, so your sources of information are going to be:

- specs
- other implementations. The WebRTC codebase is the gold standard but it's really hard to navigate, so maybe check some alternative libraries such as pion rtc. https://github.com/pion/webrtc/
